addon.json

```diff
@@ -1,6 +1,8 @@
 {
     "package-name": "djangocms-bootstrap4",
     "installed-apps": [
+        "djangocms_link",
+        "djangocms_picture",
```

Comment:
Do these need to be in installed apps if they are not going to be used directly?

Reply:
@stefanfoulis yes they need to be installed otherwise the templates are not loaded